cpp/oneapi/dal/backend/memory.hpp

```diff
@@ -213,6 +176,94 @@ inline unique_usm_ptr<T> make_unique_usm_host(const sycl::queue& q, std::int64_t
     return unique_usm_ptr<T>{ malloc_host<T>(q, count), usm_deleter<T>{ q } };
 }
 
+inline sycl::event memcpy(sycl::queue& queue,
+                          void* dest,
+                          const void* src,
+                          std::size_t size,
```

Comment:
Here is `size_t`, but in other function like `memcpy_host2usm`, `memcpy_usm2host` `int64_t` is used